docs-ref-services/mysql.md

```diff
@@ -23,7 +23,7 @@ Learn more about [Azure Database for MySQL](https://docs.microsoft.com/azure/MyS
 
 ## Client Package
 
-### Install npm module
+### Install the npm module
 
 Use npm to install the MySQL client module.
```

Comment:
@clarkio good catch, thanks. Fixed in https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/128.